docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -37,3 +37,14 @@ function humsci_basic_preprocess_field(&$variables, $hook) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function humsci_basic_preprocess_page(&$vars) {
+  // Variant setting for the global footer.
+  $gfv = theme_get_setting('global_footer_variant');
+  if ($gfv !== "none") {
+    $vars['global_footer_variant'] = 'su-global-footer--' . $gfv;
```

Comment:
could we rename this to `$vars['global_footer_variant_classname']`? I think it's clearer. You'll also need to switch that naming when you give the variable to the template on the page.html.twig :)